.github/workflows/static-analysis.yml

```diff
@@ -38,38 +37,3 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
-
-  staticcheck:
```

Comment:
What is the reason for removing staticcheck & lint jobs?

Reply:
reverted those changes.